### PR TITLE
MOTECH-2563: Added new way to install ActiveMQ.

### PR DESCRIPTION
**Pull request by mkruszynski**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by sebbrudzinski** [comment]
@PiotrZurawskiSolDevelo We need to make sure those instructions resolve the problem with Queues/Topics tab.

**Comment by PiotrZurawskiSolDevelo** [comment]
@sebbrudzinski I checked this and still I can not see Queues/Topics tab.

**Comment by sebbrudzinski** [comment]
@mkruszynski @DamianNowakSoldevelo Please work with @PiotrZurawskiSolDevelo  to investigate/repro the issue.

**Comment by PiotrZurawskiSolDevelo** [comment]
I checked this once again with help from @DamianNowakSoldevelo and when Second part of this doc is not used (I removed this), it's working correctly. In the doc should be added information about this, because both options can not work together or just remove this because it's not working.

File: docs/source/faq.rst [hunk 1]
```diff
@@ -173,22 +173,20 @@ Then connect to Tomcat using a remote debugger on port 8000, same as when normal
 Why am I not seeing anything in "Admin Queues and Topics"?
 ----------------------------------------------------------
 
-The most likely cause is a RMI connection error. By default RMI is using a random port.
-To set fixed port you have to edit ActiveMQ broker configuration, which is in the file ``/etc/activemq/instances-enabled/main/activemq.xml``.
-Add the following lines to the broker configuration (the name of the broker in use is in ``Admin/Settings/JMX/Broker name``):
+1. Edit ActiveMQ broker configuration, which is in the file ``/etc/activemq/instances-enabled/main/activemq.xml``.
+Change or add the following lines in the broker tag;
```
Comment (sebbrudzinski) [inline docs/source/faq.rst]: Make it say "change or add the following attribute to the broker element"

**Comment by sebbrudzinski** [comment]
test this please

**Pull request merged**